### PR TITLE
chore: task observability

### PR DESCRIPTION
**Pull request by ursucarina**
# TL;DR
Initial setup for task observability

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description

To setup:
* run `git fetch -p && git checkout carina/obs`
* run `make pack`
* `cd packages/console`
* `yalc publish`
* run `yarn build:watch`

Go to `cloud/clients` repo
* run `git fetch -p && git checkout carina/obs`
* run `make prep_console`
* cd `packages/zapp/console`
* run `yalc add @flyteorg/console`
* run `yarn start`

### CODE REVIEW

File: packages/console/src/components/Executions/TaskExecutionsList/TaskExecutionObservability.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  WorkflowExecutionIdentifier,
+} from 'models/Execution/types';
+import * as React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+import { useExternalConfigurationContext } from 'basics/ExternalConfigurationProvider';
+import { useNodeExecution } from 'components/hooks/useNodeExecution';
+import { WaitForData } from 'components/common/WaitForData';
+import { TaskExecutionDetailsAppBarContent } from './TaskExecutionDetailsAppBarContent';
+
+export interface ExecutionDetailsRouteParams {
```
Comment (jsonporter) [inline packages/console/src/components/Executions/TaskExecutionsList/TaskExecutionObservability.tsx:14]: I think we're going to need a "view" (enum?) here too; the links need to be able to land on a specific tab within the attempt.

**Review by jsonporter — APPROVED** [review]
